ui/appui-react/src/appui-react/stagepanels/StagePanelDef.tsx

```diff
@@ -100,9 +108,10 @@ export class StagePanelDef extends WidgetHost {
       const side = toPanelSide(this.location);
       frontstageDef.nineZoneState = setPanelSize(frontstageDef.nineZoneState, side, size);
       const panel = frontstageDef.nineZoneState.panels[side];
-      if (panel.size === this._size)
+      if (panel.size !== size) {
```

Comment:
If this is `true`, it is only because the `size` have been constrained by the max/min, I think we probably should send the change event anyway. (and not return right away)

Reply:
My code change wasn't correct here since I misunderstood how the previous code worked. I'll revert the changes. Also, I've consulted Gerardas on whether or not we should emit if the size after the resize did not change and he was against it, so for now I'll just return. Unless you know some use case or a common pattern that needs different behavior.